src/Sql/Sql.Sdk/Generated/Models/ServerBlobAuditingPolicy.cs

```diff
@@ -139,7 +139,7 @@ public ServerBlobAuditingPolicy()
         /// SELECT on SCHEMA::mySchema by public
         ///
         /// For more information, see [Database-Level Audit
-        /// Actions](https://learn.microsoft.com/en-us/sql/relational-databases/security/auditing/sql-server-audit-action-groups-and-actions#database-level-audit-actions)</param>
+        /// Actions](https://docs.microsoft.com/en-us/sql/relational-databases/security/auditing/sql-server-audit-action-groups-and-actions#database-level-audit-actions)</param>
```

Comment:
The docs URL has been updated to learn.microsoft.com instead of docs.microsoft.com. Please revert the change and re-generate the code.